src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs

```diff
@@ -122,5 +161,31 @@ public void AnalyzeDirectory_DirectoryWithOtherJsonFiles_LogsExpectedErrorInSari
             Assert.AreNotEqual(null, toolNotifications[1]["exception"]);
             Assert.AreEqual(null, toolNotifications[2]["exception"]);
         }
+
+        [DataTestMethod]
+        [DynamicData(nameof(LogSummaryScenarios), DynamicDataDisplayName = nameof(GetDisplayName))]
```

Comment:
The log summaries are only different by a single line (the verbose option).  My thought was to have this test method take a single argument `(bool verbose)` and use that to determine whether to include "--verbose" in the command and whether that one extra line should be included when building the expected summary string.